frontend/src/components/viewers/MetricsVisualizations.test.tsx

```diff
@@ -135,6 +135,12 @@ describe('ConfidenceMetricsSection', () => {
     linkedArtifacts?: LinkedArtifact[],
     fullArtifactPathMap?: FullArtifactPathMap,
   ): ConfidenceMetricsSectionProps {
+    const lineColorsStack = [...lineColors].reverse();
```

Comment:
Fix tests as initial color mappings are now set in the parent component to the `MetricsVisualizations.tsx`'s `ConfidenceMetricsSection`.